packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -118,23 +92,12 @@ export const CircularProgress = forwardRef<
     }
   }, [ariaLabel]);
 
-  const getValueProps = () => ({
-    unit,
-    value,
-    getValueProps: (valueProps = {}) => ({
-      className: withBaseName("progressValue"),
-      ...valueProps,
-    }),
-  });
-
   let progressInfo: ReactNode = null;
   if (showInfo) {
-    progressInfo = renderInfo ? (
-      renderInfo(getValueProps())
-    ) : (
-      <DefaultInfo
+    progressInfo = (
+      <Info
         className={withBaseName("progressValue")}
-        unit={unit}
+        unit="%"
         value={Math.round(progress)}
         {...rest}
```

Comment:
why would you pass {...rest} on Info and root element?

Reply:
removed from info